swift/ql/lib/codeql/swift/generated/Db.qll

```diff
@@ -0,0 +1,1500 @@
+module Db {
```

Comment:
More name bikeshedding: How about calling this `Raw` instead?

Reply:
I like that! So maybe we can also rename `from/toDbInstance` to `from/toRawInstance`?

Reply:
Sounds good. I'm not sure I like the word "instance" since this is not a word we really use in QL. Ruby calls these `fromGenerated/toGenerated`. Could we do the same maybe?

Reply:
I went with `fromRaw`/`toRaw` :slightly_smiling_face: